src/drivers/mct/shr/seq_timemgr_mod.F90

```diff
@@ -910,6 +916,7 @@ subroutine seq_timemgr_clockInit(SyncClock, nmlfile, restart, restart_file, pioi
             opt_n   = restart_n,           &
             opt_ymd = restart_ymd,         &
             RefTime = CurrTime,            &
+            cplTimeStep = cplTimeStep, &
             alarmname = trim(seq_timemgr_alarm_restart))
 
        call seq_timemgr_alarmGet(SyncClock%EAlarm(n,seq_timemgr_nalarm_restart), IntSec=RestInterval)
```

Comment:
It seems like the following 6 lines are no longer needed: from my read of the code, minRestInterval is set again, in a different way, before it is ever needed.